packages/components-library/tests/unit/filters/FilterCard.spec.ts

```diff
@@ -0,0 +1,57 @@
+import { mount, createLocalVue } from '@vue/test-utils'
+import FilterCard from '@/components/filters/FilterCard.vue'
+import { BootstrapVue } from 'bootstrap-vue'
+
+const localVue = createLocalVue()
+localVue.use(BootstrapVue)
+
+const name = 'name'
```

Comment:
Wouldn't it be easier to read if you named it like:
```javascript
const propsData = {name: 'name'}
```

And later on use destructuring, e.g.:
```javascript
const wrapper = mount(FilterCard, { localVue, {...propsData, collapsed: false })
```

Reply:
no this would break the test as the code in the second snippet is missing the property name propsData